mycli/myclirc

```diff
@@ -86,6 +86,11 @@ Token.Menu.Completions.MultiColumnMeta = 'bg:#aaffff #000000'
 Token.Menu.Completions.ProgressButton = 'bg:#003333'
 Token.Menu.Completions.ProgressBar = 'bg:#00aaaa'
 
+# Query results
+Token.Output.Header = 'bold'
+Token.Output.OddRow = ''
+Token.Output.EvenRow = ''
```

Comment:
I'm not against defaulting the colors, but I don't think we should use background colors -- those will be way too hard to get right for people's terminal color schemes. If we choose bright enough colors, it should be somewhat sane on dark and light. For example:

<img width="170" alt="screen shot 2017-06-27 at 08 53 29" src="https://user-images.githubusercontent.com/3047055/27591038-22f694ca-5b16-11e7-8c25-f244c7cca42f.png">

<img width="182" alt="screen shot 2017-06-27 at 08 52 59" src="https://user-images.githubusercontent.com/3047055/27591042-25af666a-5b16-11e7-9d1b-24eea908490e.png">

Settings:
```
Token.Output.Header = 'bold'
Token.Output.OddRow = '#ansiteal'
Token.Output.EvenRow = '#ansidarkblue'
```